src/Apps/Order/Utils/__tests__/formValidators.jest.tsx

```diff
@@ -1,5 +1,24 @@
 import { Address } from "Components/AddressForm"
-import { validateAddress, validatePresence } from "../formValidators"
+import {
+  validateAddress,
+  validatePresence,
+  validatePostalCode,
+} from "Apps/Order/Utils/formValidators"
+
+describe("formValidators/validatePostalCode", () => {
+  it("returns error when postal code is invalid for US", () => {
+    expect(validatePostalCode("XX", "US")).toBe("This field is required")
+  })
+  it("returns no error when postal code is valid for US", () => {
+    expect(validatePostalCode("15601", "US")).toBe(null)
```

Comment:
**suggestion (non-blocking):** Your US ZIP Code regex supports 9-digit ZIP Codes as well, so it would be nice to add a test to ensure that we don't lose that ability if the regex is changed in the future.

Reply:
thank you for this suggestion.